aztec/src/main/kotlin/org/wordpress/aztec/spans/MarkSpan.kt

```diff
@@ -1,12 +1,47 @@
 package org.wordpress.aztec.spans
 
+import android.graphics.Color
 import android.text.TextPaint
 import android.text.style.CharacterStyle
 import org.wordpress.aztec.AztecAttributes
+import org.wordpress.aztec.source.CssStyleFormatter
 
-class MarkSpan(override var attributes: AztecAttributes = AztecAttributes()) : CharacterStyle(), IAztecInlineSpan {
+class MarkSpan : CharacterStyle, IAztecInlineSpan {
     override var TAG = "mark"
 
-    override fun updateDrawState(tp: TextPaint?) {
+    override var attributes: AztecAttributes = AztecAttributes()
+    var textColor: Int? = null
```

Comment:
Since `textColor` is only set in the constructor we could use an immutable property. If we also rename it as suggested below this could look like:

```
private val textColorValue: Int?
```

Reply:
Thank you for the suggestion Antonis! 

Once I make this change with the [following suggestion](https://github.com/wordpress-mobile/AztecEditor-Android/pull/1073/files#r1532075080) I'm getting:

`Val cannot be reassigned`

I'm not sure if I'm missing something 🤔

Reply:
Hey @geriux 👋 
I think the error occurs since this change depends on the other suggestions. I prepared a [patch](https://github.com/wordpress-mobile/AztecEditor-Android/files/14693797/markspan.patch) with all the suggestions. Let me know if that helps.
Sorry for confusing things with my multiple inline comments.

Reply:
Ohhhh since I was doing a change at a time I was encountering that error but it was highlighting it where the variable was being declared 🙃 

Thank you for sharing the patch!